### DISCUSSION TITLE
Can `NavigationItem` be added this way?

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8119] [Help]
The `->items(...)` method exists for `NavigationGroup::`, but if I write my code the following way, the `NavigationItem` does not show in the menu. Any idea why? It is definitely more readable in this manner than to separate the items into `->navigationItems([])`

```PHP
->navigationGroups([
    NavigationGroup::make()
        ->collapsed()
        ->label('Reports')
        ->items([
            NavigationItem::make('Analytics')
                ->url('https://filament.pirsch.io', shouldOpenInNewTab: true)
                ->icon('heroicon-o-presentation-chart-line')
                ->sort(3),
        ]),
```

**Comment by danharrin** [comment]
https://filamentphp.com/docs/3.x/panels/navigation#advanced-navigation-customization
